manta-trusted-setup/Cargo.toml

```diff
@@ -45,10 +45,8 @@ std = ["manta-util/std"]
 test = ["manta-crypto/test"]
 
 [dependencies]
-ark-bn254 = { version = "0.3.0", optional = true, default-features = false, features = ["curve", "scalar_field"] }
 ark-groth16 = { version = "0.3.0", default-features = false }
 ark-poly = { version = "0.3.0", default-features = false }
-ark-relations = { version = "0.3.0", default-features = false }
```

Comment:
Why are we able to remove this?

Reply:
This is imported in `manta-crypto` already.